requirements.txt

```diff
@@ -14,7 +14,6 @@ torch-xla==2.6.0;sys_platform != 'darwin'
 # Pinned to 0.5.0 on CPU. JAX 0.5.1 requires Tensorflow 2.19 for saved_model_test.
 # Note that we test against the latest JAX on GPU.
 jax[cpu]==0.5.0
-flax
-
+flax>=0.10.1
```

Comment:
this seems to run counter to your change above, where you are explicitly installing nnx only for the nnx enabled testing you are running. if you add it here, the `pip install flax` line above does nothing, and we have no coverage for running jax tests without flax installed.

i think you probably want to remove this line.